### PR TITLE
Adds Rudolta of the Sleigh

### PR DESCRIPTION
**Pull request by gailelel**
## About The Pull Request
Adds Rudolta and the Ego required.

## Why It's Good For The Game
More abnormalities are always a good thing, and this one is a white/red HE, which we are sorely needing

## Changelog
:cl:
add: Added Rudolta and EGO
/:cl:

### CODE REVIEW

**Comment by Kitsunemitsu** [comment]
I personally walked her through this, if you have any questions/concerns, come to me and I'll bring it up.

File: code/modules/mob/living/simple_animal/abnormality/he/rudolta.dm [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	return
+
+/mob/living/simple_animal/hostile/abnormality/rudolta/Life()
+	. = ..()
+	if(!.) // Dead
+		return FALSE
+	if((pulse_cooldown < world.time) && !(status_flags & GODMODE))
+		WhitePulse()
+
+/mob/living/simple_animal/hostile/abnormality/rudolta/CanAttack(atom/the_target)
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/rudolta.dm:58]: So you want it to be able to attack people..?

Comment (gailelel) [reply code/modules/mob/living/simple_animal/abnormality/he/rudolta.dm:58]: We tested this; it's like this to follow someone around, and doesn't actually attack people. If it wasn't, then Rudolta would just stay in containment.

File: code/modules/mob/living/simple_animal/abnormality/he/rudolta.dm [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	if(ishuman(the_target))
```
Comment (EgorDinamit) [inline code/modules/mob/living/simple_animal/abnormality/he/rudolta.dm:59]: Regardless, this is absolutely pointless.
There is a variable just for that anyway, and we don't need it to target only humans anyway.

**Review by EgorDinamit — CHANGES_REQUESTED** [review]
Increase its move_to_delay to 5-6.
Make it unable to attack(see how Silent Orchestra and violet ordeals work).

**Comment by Kitsunemitsu** [comment]
Hey, Egor, So
I took this into a local server, and it works exactly as intended.
It can move around, following players, dealing AOE damage.
It does 0 damage, so it can't really "attack people"
Otherwise it just stays in it's room and does nothing.

**Pull request merged**